{"raw":{},"required":["Filename","Size"],"package":"python3-bleak-retry-connector","source":"bleak-retry-connector","version":"3.8.0-1","section":"python","priority":"optional","architecture":"all","depends":["python3-async-timeout | python3-supported-min (>= 3.11)","python3-bleak","python3-bluetooth-adapters","python3-dbus-fast","python3:any"],"installedSize":79,"maintainer":"Debian Python Team <team+python@tracker.debian.org>","description":"Connector for Bleak Clients that handles transient connection failures","homepage":"https://github.com/bluetooth-devices/bleak-retry-connector","filename":"pool/main/b/bleak-retry-connector/python3-bleak-retry-connector_3.8.0-1_all.deb","size":21208,"md5":"e0bd23b74e8bdc74009922e5d6eefafb","sha256":"42e2336cc608fbecbf4b0cb96ff5f693bd034e6d97f9355eeaafe8c6d3314c39","descriptionMd5":"94cf3179d80dc95dc44c0b30cf94cbad"}